src/components/modules/Earn/Earn.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import PageBody from "../../../components/common/PageBody";
+
+import SpendCardCollection from "./SpendCardCollection";
+import EarnPoints from "./EarnPoints";
+import CurrentPoints from "./CurrentPoints";
+
+export default function Earn() {
+  return (
+    <PageBody>
+      <br />
```

Comment:
Shouldn't be using <br /> to control page layout. We'll want to use margins and padding to control page formatting